content/en/apps/guides/hosting/vertical-vs-horizontal.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Vertical vs Horizontal scaling in CHT 4.x"
+linkTitle: "Vertical vs Horizontal Scaling"
+weight: 20
+description: >
+    The power of clustered CouchDB to horizontally scale the CHT  
+relatedContent: >  
+    apps/guides/hosting/4.x/self-hosting-multiple-nodes
+    apps/guides/hosting/4.x/data-migration
+    core/overview/architecture/
+---
+
+## Introduction
+
+CHT Core 4.0.0 introduces [a new architecture]({{< relref "core/overview/architecture" >}}) for hosting.  Specifically, 4.0.0 gives the ability to add more servers, known as horizontally scaling, so that large deployments can support more users.  This is a big improvement over naively adding more RAM and CPU to a single server where the CHT Core is running, known as vertically scaling.
```

Comment:
Is it worth specifically mentioning CouchDB servers, at least until we support horizontally scaling other services?

```suggestion
CHT Core 4.0.0 introduces [a new architecture]({{< relref "core/overview/architecture" >}}) for hosting.  Specifically, 4.0.0 gives the ability to add more CouchDB servers, known as horizontally scaling, so that large deployments can support more users.  This is a big improvement over naively adding more RAM and CPU to a single server where the CHT Core is running, known as vertically scaling.
```

Reply:
@craig-landry  gave the feedback that "CouchDB" is a bit scary when you're just trying to get your head around all this crazy scaling business.  I'll push back on this suggestion, but we can re-visit if you feel strongly!